hazelcast-jet-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/UnsupportedOperationVisitor.java

```diff
@@ -236,6 +237,7 @@ public Void visit(SqlDataTypeSpec type) {
             case DATE:
             case TIME:
             case TIMESTAMP:
+            case TIMESTAMP_WITH_LOCAL_TIME_ZONE:
```

Comment:
Looking at IMDG UOV we might want to add `SUPPORTED_KINDS.add(SqlKind.MOD);` as well